docs/topics/modules/using_stack_report.adoc

```diff
@@ -1,62 +1,62 @@
-
-* *Additional components recommended by {osio}:* A list of additional components commonly used by similar stacks.
+image::pipeline_stackreport.png[Acessing Stack Report]
++
+The report displays a summary of four key aspects relevant to your stack in the form of cards:
++
+image::full_stack_reports.png[Full Stack Report]
 +
-See <<interpreting_stack_report>> section for further details.
+Security Issues:: This card highlights the number of security issues in your stack, the highest Common Vulnerability Scoring System (CVSS) score, and the number of dependencies with this high score.
+Licenses:: This card suggests an appropriate stack level license, and flags conflicting, unknown, and restrictive licenses (licenses that are not commonly used in similar stacks or that do not work well with the stack’s representative license) affecting your stack.
+Insights:: Based on the analysis of other similar stacks, this card identifies usage outliers (dependencies not commonly used in similar stacks and that rarely go well together) in your stack and also highlights the number of companion (additional) dependencies that could augment your stack.
+Dependency Details:: This card lists the number of dependencies analyzed by {osio} and those unknown to it.
+. Click each of the cards to see <<interpreting_stack_report,detailed information>> about each of the aspects relevant to your stack.
+. Expand the arrow adjacent to the dependency to see information about:
 
-. Filter the *Detail analysis of your stack components* list to see:
+* The current and latest available version
+* Licenses, and tags associated with it
+* GitHub statistics relevant to it that help assess its popularity
 
-** Usage Outliers (image:usage_outlier.png[title="Usage Outlier"])
-** Security Issues (image:security_issues.png[title="Security Issues"])
-** Alternate Components (image:alternate_component.png[title="Alternate Components"])
-** Grouped Components (image:alternate_component.png[title="Alternate Components"])
-** License Outliers (image:license_outlier.png[title="License Outliers"])
-** Unknown Licenses (image:license_unknown.png[title="Unknown Licenses"])
-** License conflict in a Component (image:license_conflicts.png[title="License Conflicts"])
 +
-Note that {osio} recommends *Alternate Components* for your stack if it identifies license or usage outliers.
+These statistics help you compare the existing dependency with the alternate dependency and make a smart choice for your stack.
 
-. In the *Action* column adjacent to the recommended alternate or additional components, click *Create Work Item* to create a work item in the {osio} Planner and <<working_with_an_existing_work_item,follow through>> on the recommendations.
+. To act on the analytics and insights provided by the report:
+* In the Security Issues detailed view, click *Report an issue* to report a security vulnerability highlighted in the report as an issue in the {osio} planner.
```

Comment:
-> `*Security Issues* detailed view`

-> Planner?

Reply:
Leaving 'planner' as is, cause Planner is not used in the UX, similarly analytics is not used anywhere, so referring to it as {osio} analytics.